pkg/summarizer/summary.go

```diff
@@ -571,6 +571,13 @@ func buildFailLink(testID, target string) string {
 }
 
 // overallStatus determines whether the tab is stale, failing, flaky or healthy.
+//
+// Tabs are:
```

Comment:
In this example, this helps a whole lot. It tells me that you're trying to cover for 5 cases, and a bit about what each one means. Then, looking at the function, I see that 3 of the 5 are handled immediately, so most of the function is looking for failing cells in recent columns. Nice.

If I'm looking to see what "Flaky" means, I _might_ stop here, and that's maybe a risk, but I can also dig into the code to figure out how "recent" works or if I think it's broken.

Reply:
The thing is, I do not want anyone maintaining this code who "stops there." Maintainers need to read the code, and documentation makes it easier for people to not do this.